tools/clang/test/CodeGenSPIRV/rich.debug.texture.hlsl

```diff
@@ -5,25 +5,24 @@
 //CHECK: [[name_cb_arr:%\d+]] = OpString "@type.cube.image.array"
 
 //CHECK: [[info_none:%\d+]] = OpExtInst %void [[ext:%\d+]] DebugInfoNone
-//CHECK: [[t3_ty:%\d+]] = OpExtInst %void [[ext]] DebugTypeComposite [[name_2d_arr]] Class [[src:%\d+]] 0 0 [[cu:%\d+]] {{%\d+}} [[info_none]]
+//CHECK: [[t3_comp:%\d+]] = OpExtInst %void [[ext]] DebugTypeComposite [[name_2d_arr]] Class [[src:%\d+]] 0 0 [[cu:%\d+]] {{%\d+}} [[info_none]]
```

Comment:
This is used to describe `Texture2DArray<S> t3;` in the line 38.
We have to create DebugTypeTemplate whose DebugTypeTemplateParameter is `S` and whose template is `Texture2DArray`.
`[[t3_comp:%\d+]] = OpExtInst %void [[ext]] DebugTypeComposite [[name_2d_arr]] Class [[src:%\d+]] 0 0 [[cu:%\d+]] {{%\d+}} [[info_none]]` is used to describe `Texture2DArray`.

Since it is a builtin type of HLSL, I use `0` for its line and column because we do not have its actual definition in the code.
I have a plan to propose this issue for the spec update. We have to determine how to describe a built-in type for its source, compilation unit, and line/column.

Reply:
Oh I see. Thanks